style.css

```diff
@@ -0,0 +1,63 @@
+body {
+  font-family: Roboto;
+}
+
+.calculator{
+  width: 248px;
+  height: 380px;
+  line-height: 1.5;
+  position: absolute;
+  top: 50%;
+  left: 50%;
+  transform: translate(-50%, -50%);
+  margin: auto;
+  background-color: #adadad;
+  align-items: center;
+  border-radius: 10px;
+  box-shadow: 10px 10px 10px 5px grey;
+
+}
+
+
+.calculator #screen {
+  width: 248px;
+  height: 70px;
+  background-color: #adadad;
+  border-radius: 10px;
+  color: #f4f4f4;
+  text-align: right;
+  font-size: 60px;
+}
+.calculator .button{
+  display: inline-block;
+  position: relative;
+  float: left;
```

Comment:
Are you sure you need both `float: left` and `display: inline-block`? Those are generally different solutions to the "getting elements to flow inline and be blocks" problem. Another solution would be flexbox.